modules/gapi/include/opencv2/gapi/core.hpp

```diff
@@ -1720,6 +1759,86 @@ GAPI_EXPORTS GMat warpAffine(const GMat& src, const Mat& M, const Size& dsize, i
                              int borderMode = cv::BORDER_CONSTANT, const Scalar& borderValue = Scalar());
 //! @} gapi_transform
 
+/** @brief Parses output of SSD network.
+Extracts detection information (box, confidence, label) from SSD output and
+filters it by given confidence and label.
+
+@note Function textual ID is "org.opencv.core.parseSSDWL"
+
+@param in Input CV_32F tensor with {1,1,N,7} dimensions.
+@param in_sz Size to project detected boxes to (size of the input image).
+@param confidence_threshold Confidence threshold, if confidence of the
+detection is smaller than this threshold, detection is rejected.
+@param filter_label If provided (!= -1), only detections with
+given label will get to the output.
+@return a vector of detected boxes and a vector of appropriate labels.
+*/
+GAPI_EXPORTS core::GDetections parseSSD(const GMat& in,
+                                        const GOpaque<Size>& in_sz,
+                                        const float& confidence_threshold = 0.5f,
+                                        const int&   filter_label = -1);
+
+/** @overload
+Extracts detection information (box, confidence, label) from SSD output and
+filters it by given confidence and by going out of bounds.
+
+@note Function textual ID is "org.opencv.core.parseSSD"
+
+@param in Input CV_32F tensor with {1,1,N,7} dimensions.
+@param in_sz Size to project detected boxes to (size of the input image).
+@param confidence_threshold Confidence threshold, if confidence of the
+detection is smaller than this threshold, detection is rejected.
+@param filter_out_of_bounds If provided true, out-of-bounds boxes are filtered.
+@return a vector of detected boxes.
+*/
+GAPI_EXPORTS core::GRects parseSSD(const GMat& in,
+                                   const GOpaque<Size>& in_sz,
+                                   const float& confidence_threshold = 0.5f,
+                                   const bool& filter_out_of_bounds = false);
+
+/** @brief Parses output of Yolo network.
+Extracts detection information (box, confidence, label) from Yolo output,
+filters it by confidence and performs non-maximum supression for overlapping boxes.
+
+@note Function textual ID is "org.opencv.core.parseYolo"
+
+@param in Input CV_32F tensor with {1,13,13,N} dimensions,
+N should satisfy
+\f[N = (\texttt{num_classes} + 5) * 5\f],
+where num_classes - a number of classes Yolo network was trained with.
+@param in_sz Size to project detected boxes to (size of the input image).
+@param confidence_threshold If confidence of the
+detection is smaller than confidence threshold, detection is rejected.
+@param nms_threshold Non-maximum supression threshold which controls minimum
+relative box intersection area required for rejecting the box with a smaller confidence.
+If 1.f, nms is not performed and no boxes are rejected.
+@param anchors Anchors Yolo network was trained with.
+@return a vector of detected boxes and a vector of appropriate labels.
```

Comment:
`appropriate` -> `respective` or `matching`, I think

Reply:
Is it synonyms? ->matching.